src/components/contact/assets/socialMedia.scss

```diff
@@ -0,0 +1,16 @@
+@import '~theme/base/variables';
+
+.socialMedia {
+  display: flex;
+  margin-top: 20px;
+  flex-flow: row-reverse;
+  .Icon__Wrapper {
```

Comment:
You can move these classes outside of `. socialMedia`, with css modules enabled we don't need to nest classes, it will help to keep the final css file smaller.